frontend/src/lib/components/CommandBar/SearchResult.tsx

```diff
@@ -69,7 +71,13 @@ export const SearchResult = ({ result, resultIndex, focused, keyboardFocused }:
             ref={ref}
         >
             <div className="px-2 py-3 w-full space-y-0.5 flex flex-col items-start">
-                <span className="text-muted-3000 text-xs">{tabToName[result.type]}</span>
+                <span className="text-muted-3000 text-xs">
+                    {result.type !== 'group'
+                        ? tabToName[result.type]
+                        : `${capitalizeFirstLetter(
+                              aggregationLabel((result.extra_fields as Group).group_type_index).plural
```

Comment:
Is there a reason we need to cast here given we have the `Group` specified in https://github.com/PostHog/posthog/pull/18965/files#diff-ed52228227f4ce03c0bf7b8aa5fcdf749a4fea0b96b4f9b6467b04f3e85a6515R22

Reply:
Good catch! This was actually a "bug" in the `SearchableEntity` preventing the discriminating union to resolve accordingly. Fix coming in.